src/optimal_retention.rs

```diff
@@ -952,7 +952,7 @@ mod tests {
         let (_, _, learn_cnt_per_day, _) =
             simulate(&config, &DEFAULT_PARAMETERS, 0.9, None, Some(cards))?;
 
-        assert_eq!(learn_cnt_per_day.to_vec(), vec![1, 3, 3]);
```

Comment:
If you have 9 review cards on a day and review limit of 10, would it not give you 1 new card?

Reply:
Nope. The limits are separated.